From 3eb9e7fb33b6cb6fc2a37d0c5fb4f5d8b9f0a02e Mon Sep 17 00:00:00 2001
From: Debarshi Ray <debarshir@gnome.org>
Date: Thu, 7 Jun 2018 15:01:37 +0200
Subject: [PATCH] app: Avoid braced initialization

RHEL 7.6 has an older GCC (gcc-4.8.5-34.el7) that doesn't like it:
  app.cc: In member function 'GdkRGBA Options::get_color_bg() const':
  app.cc:310:39: error: cannot convert 'const GdkRGBA {aka const
    _GdkRGBA}' to 'gdouble {aka double}' in initialization
    GdkRGBA color{bg_color};
                          ^

https://bugzilla.redhat.com/show_bug.cgi?id=1569801
---
 src/app/app.cc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/app/app.cc b/src/app/app.cc
index 3faa2d76dd10..56b00ac49399 100644
--- a/src/app/app.cc
+++ b/src/app/app.cc
@@ -307,7 +307,7 @@ public:
                 else
                         alpha = get_alpha();
 
-                GdkRGBA color{bg_color};
+                GdkRGBA color = bg_color;
                 color.alpha = alpha;
                 return color;
         }
-- 
2.14.4

